applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -505,3 +518,16 @@ def _create_line_search_strategy(self):
                                                      self.settings["move_mesh_flag"].GetBool())
         strategy.SetUseOldStiffnessInFirstIterationFlag(self.settings["use_old_stiffness_in_first_iteration"].GetBool())
         return strategy
+
+    def _create_arc_length_strategy(self):
+        settings = self.settings["solving_strategy_settings"]["advanced_settings"]
+        settings.AddValue("max_iteration", self.settings["max_iteration"])
+        settings.AddValue("compute_reactions", self.settings["compute_reactions"])
+        settings.AddValue("reform_dofs_at_each_step", self.settings["reform_dofs_at_each_step"])
+        settings.AddValue("move_mesh_flag", self.settings["move_mesh_flag"])
```

Comment:
in the future we will also go in that direction for the other strategies, but this should be in a different PR :)

Reply:
Agree